### PR TITLE
[pt-br] Add docs/tasks/access-application-cluster/port-forward-access-application-cluster.md

### PR DESCRIPTION
**Pull request by ricmmartins**
Adding content/pt-br/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md

### CODE REVIEW

File: content/pt-br/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+   mongo-75f59d57f4   1         1         1       3m12s
+   ```
+
+2. Crie um serviço (*Service*) para expor o MongoDB na rede:
+
+   ```shell
+   kubectl apply -f https://k8s.io/examples/application/mongodb/mongo-service.yaml
+   ```
+
+   A saída (output) de um comando bem-sucedido verifica que o serviço foi criado:
```
Comment (devlware) [inline content/pt-br/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md:79]: you don´t need the word output here.

File: content/pt-br/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+Esta página mostra como usar o `kubectl port-forward` para se conectar a um servidor MongoDB em execução em um cluster Kubernetes. Esse tipo de conexão pode ser útil para depuração de bancos de dados.
+
+## {{% heading "prerequisites" %}}
+
+* {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+* Instale o [MongoDB Shell](https://www.mongodb.com/try/download/shell).
+
+<!-- steps -->
+
+## Criando a implantação (*deployment*) e o serviço (service) do MongoDB
```
Comment (devlware) [inline content/pt-br/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md:19]: I don´t think we need to have both 'deployment' and 'service' referenced when there is the translation. I asked this in our pt slack channel.

File: content/pt-br/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+
+   Uma solicitação de ping bem-sucedida retorna:
+
+   ```
+   { ok: 1 }
+   ```
+
+### Opcionalmente, deixe kubectl escolher a porta local {#let-kubectl-choose-local-port}
+
+Se você não precisa de uma porta local específica, pode permitir que o `kubectl` escolha e aloque a porta local e, assim, se livrar da necessidade de gerenciar conflitos de porta local, com a sintaxe ligeiramente mais simples:
```
Comment (devlware) [inline content/pt-br/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md:178]: Can we change aloque to reserve and maybe "se livrar" to something that uses the word evitar?

**Review by devlware — COMMENTED** [review]
Thanks for helping Ricardo.

**Review by devlware — COMMENTED** [review]
/lgtm

**Review by jcjesus — APPROVED** [review]
Thanks @ricmmartins, great job!

**Pull request merged**